feature-find-meal/src/main/java/com/kinandcarta/feature/find/meal/view/MapsFragment.kt

```diff
@@ -35,12 +40,34 @@ class MapsFragment : Fragment() {
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-        return inflater.inflate(R.layout.fragment_maps, container, false)
+        val view: View = inflater.inflate(R.layout.fragment_maps, container, false)
+        setupUI()
+        bottomSheetBehavior = BottomSheetBehavior.from(view.findViewById(R.id.list_bottomSheet))
+        bottomSheetBehavior.addBottomSheetCallback(object :
+            BottomSheetBehavior.BottomSheetCallback() {
+
+            override fun onSlide(bottomSheet: View, slideOffset: Float) {
+                // handle onSlide
+            }
+
+            override fun onStateChanged(bottomSheet: View, newState: Int) =
+                when (newState) {
+                    BottomSheetBehavior.STATE_COLLAPSED -> Toast.makeText(context, "STATE_COLLAPSED", Toast.LENGTH_SHORT).show()
```

Comment:
Not sure if we want to keep the Toast showing state changes on the bottomsheet now ?

Reply:
Just commented about this!